main/src/cgeo/geocaching/connector/gc/GCParser.java

```diff
@@ -1410,10 +1410,17 @@ static Trackable parseTrackable(final String page, final String possibleTracking
 
         // trackable owner name
         try {
-            final MatcherWrapper matcherOwner = new MatcherWrapper(GCConstants.PATTERN_TRACKABLE_OWNER, page);
-            if (matcherOwner.find()) {
-                trackable.setOwnerGuid(matcherOwner.group(1));
-                trackable.setOwner(matcherOwner.group(2).trim());
+            final MatcherWrapper matcherOwnerNew = new MatcherWrapper(GCConstants.PATTERN_TRACKABLE_OWNER_NEW, page);
+            if (matcherOwnerNew.find()) {
+                trackable.setOwnerGuid(matcherOwnerNew.group(1));
+                trackable.setOwner(matcherOwnerNew.group(2).trim());
+
+            } else {
+                final MatcherWrapper matcherOwner = new MatcherWrapper(GCConstants.PATTERN_TRACKABLE_OWNER, page);
```

Comment:
Do we need this fallback? From my testings the profile URLs were changed for all Trackables/Caches in one step. So no need for if/else which increases code complexity...

Reply:
And even if needed, it could be combined with `https?://www.geocaching.com/p(rofile)?/` (attention: in this way this will change the group indexes)

Reply:
Yeah, thought the same after reading your first comment. Not sure whether the change is rolled out completely yet, but with the adapted pattern we are on the safe side.